std_msgs/Header header
string group_name
bool is_absolute
string axis  # Could be x y z roll pitch yaw
float64 goal
float64 tolerance
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status